python/lsst/obs/ctio0m9/ingest.py

```diff
@@ -107,7 +168,15 @@ def _translateFromCalibId(self, field, md):
         return match.groups()[0]
 
     def translate_filter(self, md):
+        print('\nMerlin says there might be a problem here\nCheck what is happening when you see this\n')
```

Comment:
This should _never_ have been left in! This was just a note-to-self, and has been removed. Apologies.